app/src/main/java/org/simple/clinic/teleconsultlog/prescription/medicines/TeleconsultMedicinesInit.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.teleconsultlog.prescription.medicines
+
+import com.spotify.mobius.First
+import com.spotify.mobius.Init
+import org.simple.clinic.mobius.first
+
+class TeleconsultMedicinesInit : Init<TeleconsultMedicinesModel, TeleconsultMedicinesEffect> {
+
+  override fun init(model: TeleconsultMedicinesModel): First<TeleconsultMedicinesModel, TeleconsultMedicinesEffect> {
+    return first(model, LoadPatientMedicines(model.patientUuid))
```

Comment:
No. When we add new medicines we open the medicines screen, when we come back to this screen we need to load the medicines or else it will only show the medicines from the model.

Reply:
Ah, gotcha.